bitnami/sonarqube/templates/deployment.yaml

```diff
@@ -78,8 +78,12 @@ spec:
             - name: provisioning
               mountPath: {{ .Values.provisioningFolder }}/extensions/plugins/
               subPath: extensions/plugins
+          {{- if .Values.plugins.resources }}
           resources: {{- toYaml .Values.plugins.resources | nindent 12 }}
+          {{- else if ne .Values.plugins.resourcesPreset "none" }}
+          resources: {{- include "common.resources.preset" (dict "type" .Values.plugins.resourcesPreset) | nindent 12 }}
           securityContext: {{- .Values.plugins.containerSecurityContext | toYaml | nindent 12 }}
```

Comment:
This may be a typo
`securityContext` should not be conditioned to `plugins.resources`